test/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.reject
```

Comment:
You don't have test run, why do you need to ignore `.reject` files?

Reply:
Fixed